strings/src/main/res/values/strings.xml

```diff
@@ -117,6 +117,7 @@
     <string name="add_repeat">Add</string>
     <string name="dont_add_repeat">Do not add</string>
     <string name="add_another_menu">Add another</string>
+    <string name="add_another_question">Add another?</string>
```

Comment:
Not sure how translations are done. Did not see it in the readme also. Kindly advice.